src/hw4_exceptions/src/main/java/com/company/hw/books/Encyclopedia.java

```diff
@@ -0,0 +1,30 @@
+package com.company.hw.books;
+
+import com.company.hw.Library;
+import com.company.hw.interfaces.NotArtistic;
+
+public class Encyclopedia  extends Library implements NotArtistic {
+
+    public Encyclopedia(String author, String name, int year, boolean avail){
+        super(author, name, year, avail);
+    }
+
+    @Override
+    public void noTyping(boolean type) {
+        if (!type) {
+            System.out.println("Our Encyclopedia must be artistic! You need to change type");
+        } else {
+            System.out.println("Our Encyclopedia isn't artistic! You are right");
+        }
+    }
+
+    @Override
+    public void howMuchPages(int pages) {
```

Comment:
howMuchPages -> no question in naming 
int getNumberOfPages()
void printExpressions()
etc...

Reply:
Fixed